kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -50,7 +53,13 @@ def check(self, _):
             # Unlike consumer offsets, fail immediately because we can't calculate consumer lag w/o highwater_offsets
             raise
 
-        total_contexts = len(self.client._consumer_offsets) + len(self.client._highwater_offsets)
+        # Report the metrics
```

Comment:
To reviewer: I moved up getting the `dicts` since `total_contexts` depends on the values of consumer and highwater offsets.